app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -379,7 +394,17 @@ export const sortListings = (
   results.sort((a: RailsRentalListing, b: RailsRentalListing) =>
     new Date(a.Lottery_Results_Date) < new Date(b.Lottery_Results_Date) ? 1 : -1
   )
-  return { open, upcoming, results, additional }
+  fcfsSales.sort((a: RailsSaleListing, b: RailsSaleListing) => {
```

Comment:
**note:** Here is the order that results from this sort:
<img width="806" alt="Screenshot 2024-11-07 at 11 34 19 AM" src="https://github.com/user-attachments/assets/6b84d5d6-d3ae-4989-b8ce-a3c89a07695d">

Reply:
**issue:** I am seeing closed fcfs listings showing up on the directory page which the ACs on the ticket mentions they should not. Can we also add some tests for this logic?

Reply:
Yep we talked through that yesterday at refinement, which resulted in that clarification